packages/admin/admin-icons/package.json

```diff
@@ -56,5 +35,26 @@
     "publishConfig": {
         "access": "public",
         "registry": "https://registry.npmjs.org"
-    }
+    },
+    "repository": {
+        "directory": "packages/admin/admin-icons",
+        "type": "git",
+        "url": "https://github.com/vivid-planet/comet"
+    },
+    "scripts": {
+        "build": "run-s clean generate-icons && run-p build:babel build:types && eslint --fix src/generated/",
```

Comment:
eslint in the package made problems because of esm. i removed the eslint  runtime dependency of the generate-icons script and moved the eslint autofix after the generation step.